guides/bulk.md

```diff
@@ -0,0 +1,204 @@
+# Bulk
+
+In this guide, you'll learn how to use the OpenSearch Golang Client API to perform bulk operations. You'll learn how to index, update, and delete multiple documents in a single request.
+
+## Setup
+
+First, create a client instance with the following code:
+
+```go
+package main
+
+import (
+    "github.com/opensearch-project/opensearch-go/v2"
+    "log"
+)
+
+func main() {
+    client, err := opensearch.NewDefaultClient()
+    if err != nil {
+        log.Printf("error occurred: [%s]", err.Error())
```

Comment:
Should we bailing here in samples, e.g. `log.Fatalf`?

Reply:
I was guided by the rules from [issue](https://github.com/opensearch-project/opensearch-go/issues/278#issue-1654505429), there was written not to throw errors, or did I misunderstand?

Reply:
Maybe, I don't know what's best. 

What's the user experience when `NewDefaultClient()` returns an error with the code as written?

Reply:
In the console there will be a reason why the client was not created and the program will complete successfully, I mean there will be no fatal termination. The user, having read the reason, understands that he did something wrong, and decides the origin reason and tries again.

Reply:
In this example the code doesn't do anything else, however in the example below:

```go
movies := "movies"
books := "books"
createMovieIndex, err := client.Indices.Create(movies)
if err != nil {
    log.Printf("error occurred: [%s]", err.Error())
}
log.Printf("response: [%+v]", createMovieIndex)
createBooksIndex, err := client.Indices.Create(books)
if err != nil {
    log.Printf("error occurred: [%s]", err.Error())
}
log.Printf("response: [%+v]", createBooksIndex)
```

You get an error the first time on  `client.Indices.Create(movies)`, then you get an error again the second time on `client.Indices.Create(books). I believe we shouldn't be bailing after the first error, no?

Reply:
I agree about the first one, but again I will refer to the guide ... It's not difficult for me to change it for a fatal mistake. At the expense of the second, at the end of each guide there are cleaning topics, and the previously created modules were cleaned there